package/MDAnalysis/core/topologyattrs.py

```diff
@@ -519,14 +519,14 @@ def set_segments(self, sg, values):
         raise NotImplementedError
 
     @classmethod
-    def is_value_missing(self, value):
+    def are_values_missing(self, values):
         """check if an attribute has a missing value
         .. versionadded:: 2.7.0
         """
         if hasattr(self, 'missing_value_label'):
-            return value == self.missing_value_label
+            return (np.array(values) == self.missing_value_label)
```

Comment:
Perhaps `np.asarray`? (https://numpy.org/doc/stable/reference/generated/numpy.asarray.html) slightly more efficient than `np.array`, and I suspect most values passed to this funciton will be arrays originally :)

Also, slight nitpick -- it's more pythonic to not have parentheses surrounding this statement.

Reply:
Could you please also add a test for this method?